packages/appstore/src/components/modals/wallets-upgrade-modal/wallets-upgrade-modal.scss

```diff
@@ -3,36 +3,55 @@
         width: 100%;
         display: flex;
         flex-direction: column;
-        align-items: flex-start;
-        gap: 1.6rem;
-        flex-shrink: 0;
-        padding: 2.4rem;
+        align-items: center;
+        gap: 3.2rem;
+        max-width: 63.8rem;
+        margin: 0 auto;
+        padding: 8rem 0;
 
         @include mobile {
-            align-items: center;
-            padding: 1.6rem;
+            gap: 1.6rem;
+            max-width: 28rem;
+            padding-top: 4.8rem;
+            padding-bottom: 1.6rem;
         }
     }
 
-    &__description {
-        width: 35.2rem;
-        display: flex;
-        flex-direction: column;
-        gap: 0.8rem;
+    &__media-container {
+        width: 56rem;
+        height: 32rem;
+        border-radius: $BORDER_RADIUS;
+        filter: drop-shadow(0px 4px 6px rgba(14, 14, 14, 0.03)) drop-shadow(0px 12px 16px rgba(14, 14, 14, 0.08));
 
         @include mobile {
-            width: 100%;
+            width: 28rem;
+            height: 16rem;
         }
     }
 
-    &__image {
-        position: absolute;
-        right: -8rem;
-        top: 0;
+    &__video {
+        width: 100%;
+        border-radius: $BORDER_RADIUS;
```

Comment:
Don't think we need this border radius.
```suggestion
```

Reply:
We do need the border radius however I feel that I can remove the one in `wallets-upgrade-moda__media-container`